File: Code TranslaterTests/Scripts/Python/1.py
import numpy as np

def normalize(x, newLowerBound, newUpperBound):
    min = np.min(x)
    max = np.max(x)
    range = max - min
    newRange = newUpperBound - newLowerBound

    return [((a - min) / range) * newRange + newLowerBound for a in x]

File: Code TranslaterTests/Scripts/Python/1/1.cs

static IEnumerable<float> normalize(IEnumerable<float> x, float newLowerBound, float newUpperBound)
{
	float min = Enumerable.Min(x);
	float max = Enumerable.Max(x);
	float range = max - min;
	float newRange = newUpperBound - newLowerBound;
	
	return x.Select(a => ((a - min) / range) * newRange + newLowerBound);
}